google-account-plugin/src/com/google/cloud/tools/intellij/login/ui/GoogleLoginAction.java

```diff
@@ -69,15 +65,14 @@ public void update(AnActionEvent e) {
       if (image == null) {
         presentation.setIcon(GoogleLoginIcons.DEFAULT_USER_AVATAR);
```

Comment:
I guess I'm just asking of we can get high dpi versions of the resources instead of scaling.